src/storage/file_io.h

```diff
@@ -366,8 +355,15 @@ struct fileio_backup_db_buffer
 typedef struct file_zip_page FILEIO_ZIP_PAGE;
 struct file_zip_page
 {
-  lzo_uint buf_len;		/* compressed block size */
-  lzo_byte buf[1];		/* data block */
+  int buf_len;			/* compressed block size */
```

Comment:
lzo_uint is 64bit long integer, int buf_len might compromise backward compatibility.

Reply:
Somewhat related to my comment above, areas allocated, whose size are saved to the buf_len are equal to size of decompressed stuff. They should not cause any problems and overflow 32bit ints.

Reply:
Ok, I understood.